src/config/ConfigManager.cpp

```diff
@@ -35,8 +40,10 @@ CConfigManager::CConfigManager() {
                 parseError += "Config error at line " + std::to_string(linenum) + ": Line parsing error.";
             }
 
-            if (!parseError.empty() && parseError.find("Config error at line") != 0) {
+            if (!parseError.empty()) {
+                // If an error there is, user may want the precise line where it occured
                 parseError = "Config error at line " + std::to_string(linenum) + ": " + parseError;
```

Comment:
if you want to log all errors, you could directly `Debug::log()` here. Works ok with hyprpaper, it's CLI anyways.

Reply:
you resolved it without doing anything and 0 comments...?